scipy/stats/mstats_basic.py

```diff
@@ -637,9 +638,9 @@ def kendalltau(x, y, use_ties=True, use_missing=False, method='auto'):
         elif n == 2:
             prob = 1.0
         elif c == 0:
-            prob = 2.0/np.math.factorial(n)
+            prob = 2.0/float_factorial(n)
```

Comment:
Should we just use `scipy.special.gamma(n+1)`?

Reply:
`math.factorial` is faster than `gamma` for most values of `n`.